templates/shared/forms/interactive/managed-apps.html

```diff
@@ -3,9 +3,24 @@
     <header class="p-modal__header" style="display: block; border-bottom: 0;">
       <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
       <h2 class="p-modal__title u-sv1" id="modal-title">Canonical, the managed app experts</h2>
-      <p>No matter where they run, we can manage your applications so you can run your business. Share with us some detail on your project and let's get started.</p>
+      <p>Canonical&rsquo;s app engineers will provide a free quote and architecture assessment &mdash; start by sharing the details below.</p>
```

Comment:
I think this `p` should be inside `.js-pagination--1` - as it is the bottom border looks too close to the paragraph:

![Screenshot from 2020-06-23 09-28-07](https://user-images.githubusercontent.com/2376968/85379963-43e17f80-b534-11ea-8c76-43c8154d5555.png)

I checked /desktop/developers#get-in-touch to see what we do there, and it looks a lot nicer:

![Screenshot from 2020-06-23 09-29-56](https://user-images.githubusercontent.com/2376968/85380022-56f44f80-b534-11ea-9685-c026991e8901.png)

So moving the `p` above the `.js-formfield` on line 9 would give you:

![Screenshot from 2020-06-23 09-29-33](https://user-images.githubusercontent.com/2376968/85380115-725f5a80-b534-11ea-953b-706f73c65a68.png)